src/components/emptyStatePages/NoOrgApplicationLandingPage.tsx

```diff
@@ -54,6 +57,7 @@ export default function NoOrgApplicationLandingPage(): JSX.Element {
         toastSuccess(strings.SUCCESS);
       }
       goToApplication(result.data);
+      reloadOrganizations();
```

Comment:
Should this happen before `goToApplication`? I guess it's all async anyway so maybe it doesn't matter.

Reply:
It is all async like you said. I think there was a problem with the OrgRouter refreshing before the redirect to applications, which was the cause of the issue before. This is now in the same effect so it should not happen. I would prefer to keep it this way just to show an order of operations logically speaking.